cmd/ovirt-csi-driver/ovirt-csi-driver.go

```diff
@@ -6,6 +6,7 @@ import (
 	"os"
 	"time"
 
+	ovirt "github.com/ovirt/csi-driver/internal"
```

Comment:
Lets do proper packaging then. 
Move the code another level down into 'ovirt' package, so internal/ovirt/.
Then we don't need those named imports.
After you do that, change the function name from NewOvirtClient to simply NewClient since the package represents ovirt already.

Reply:
Done